python/tank/deploy/tank_commands/setup_project_wizard.py

```diff
@@ -641,4 +641,4 @@ def execute(self):
             core_localize.do_localize(self._log, 
                                       self._params.get_configuration_location(sys.platform), 
                                       suppress_prompts=True,
-                                      strip_toolkit_credentials=self._is_session_based_authentication_supported())
```

Comment:
This was the only instance in the whole codebase where we assigned something other that False to that parameter.